templates/fonts/swift4.stencil

```diff
@@ -64,8 +63,13 @@
     {% for font in family.fonts %}
     {{accessModifier}} static let {{font.style|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}} = FontConvertible(name: "{{font.name}}", family: "{{family.name}}", path: "{% call transformPath font.path %}")
     {% endfor %}
+    {{accessModifier}} static let all: [FontConvertible] = [{% for font in family.fonts %}{{font.style|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}}{% if not forloop.last %}, {% endif %}{% endfor %}]
   }
   {% endfor %}
+  {{accessModifier}} static let allCustomFonts: [FontConvertible] = [{% for family in families %}{{family.name|swiftIdentifier:"pretty"|escapeReservedKeywords}}.all{% if not forloop.last %}, {% endif %}{% endfor %}].flatMap { $0 }
+  {{accessModifier}} static func registerAllCustomFonts() {
+    allCustomFonts.forEach { $0.register() }
```

Comment:
I almost never use `forEach`, always `for ... in ...` for some reason.

Reply:
Probably a remnant from my Objective-C days, where `for ... in` was more performant than the block based `forEach`.

Reply:
Yeah I think in Swift that's the opposite, `forEach` might be optimised while `for … in` might not (because `for … in` can break? tomato tomato though